Networking/Networking/Remote/StripeRemote.swift

```diff
@@ -46,6 +59,7 @@ private extension StripeRemote {
     enum Path {
         static let connectionTokens = "wc_stripe/connection_tokens"
         static let accounts = "wc_stripe/account/summary"
+        static let locations = "payments/terminal/locations/store"
```

Comment:
Is this path correct? See https://github.com/woocommerce/woocommerce-gateway-stripe/blob/5313c58805fe0531e7de6caf298471d9c2d7b4af/includes/admin/class-wc-rest-stripe-locations-controller.php#L18

Reply:
I think it is: https://github.com/woocommerce/woocommerce-gateway-stripe/blob/5313c58805fe0531e7de6caf298471d9c2d7b4af/includes/admin/class-wc-rest-stripe-locations-controller.php#L68

Reply:
@ctarda Shouldn't the path start with `wc_stripe` instead of `payments` => `/wc_stripe/terminal/locations/store`?

Reply:
Can confirm - just tested now - it should definitely be "wc_stripe/terminal/locations/store"

Reply:
Whoopsies!